.gitignore

```diff
@@ -14,3 +14,5 @@
 # Ignore all logfiles and tempfiles.
 /log/*.log
 /tmp
+.DS_Store
```

Comment:
to stop adding this to every project gitignore you work on, consider adding this to a global gitignore.